lib/core/kotekanMode.cpp

```diff
@@ -88,6 +89,13 @@ void kotekanMode::initalize_stages() {
     // Apply config for Telescope class
     Telescope::instance(config);
 
+    // Create and register kotekan trackers before stages created
+    KotekanTrackers::instance().register_with_server(&restServer::instance());
+    if (config.get_default<bool>("/trackers_dump", "enabled", false)) {
```

Comment:
I would put this logic in the constructor, and pass `config` to the constructor like the telescope and datasetManager objects.